registrations.md

```diff
@@ -30,15 +29,15 @@ In addition to node registration, there is an additional step of pledging a stak
 The sequence for node registration is shown in the following diagram:
 ![aggregator staking](./images/aggregator-stake.png)
 
-These are the steps to become an aggregator.
-* Register with the L1 Network Management contract and pay a significant stake in the Obscuro token. The stake has multiple roles. The first one is to penalize aggregators who attempt to hack the protocol, and second is for the aggregators to buy into the ecosystem, so that they will make an effort to keep it running smoothly.
+These are the steps to becoming an Aggregator.
+* Register with the L1 Network Management contract and pay a significant stake in the Obscuro token. The stake has multiple roles. The first one is to penalise Aggregators who attempt to hack the protocol, and the second is for the Aggregators to buy into the ecosystem and make an effort to keep it running smoothly.
 * Set up a server with a valid, unaltered, up-to-date and secured TEE and provide an attestation from the hardware manufacturer or a delegate to the management contract.
-* On seeing this request to join the network published to the L1, another registered TEE will share the shared secret, used to encrypt and decrypt user transactions.
-* Once in possession of the secret, the TEE can start processing all the L2 transactions that are stored on the L1 blockchain and build the state.
-* Once this is completed, the new aggregator can join the gossip with the other aggregators and participate in the lottery for producing rollups.
-* Some end users will send encrypted instructions directly to this server, and it will gossip these with other nodes in the L2 network, encrypted with the shared secret.
-* As aggregators process messages, they maintain the L2 state in the encrypted TEE memory or encrypted in a local database. If they are the winner of the lottery, when the time comes they will create a valid rollup and publish it to L1.
-* All aggregators keep track of the blocks submitted to the management contract to make sure they are up-to-date with the source of truth.
-* The first aggregator to register has a special role, as it has to create the shared secret.
+* On seeing this request to join the network published to the L1, another registered TEE shares the secret to encrypt and decrypt user transactions.
+* Once in possession of the secret, the TEE can start processing all the L2 transactions stored on the L1 blockchain and build the state.
+* Upon completion, the new aggregator can join the gossip with the other aggregators and participate in the lottery for producing rollups.
+* Some end users send encrypted instructions directly to this node server. The node gossips these with other nodes in the L2 network, encrypted with the shared secret.
```

Comment:
"this node server" << this is the first (and only?) time we refer to a node **server**. Remove "server"?

Reply:
I think it's because it's in the context of "set up a server with a valid, unaltered..." and the line "node server" makes the transition from it being just a server to being a node, the next time we talk about it, it's in the context of it being a node.

@tudor-malene is that right?